crypto/x509/x509_vpm.c

```diff
@@ -306,6 +306,17 @@ unsigned long X509_VERIFY_PARAM_get_flags(X509_VERIFY_PARAM *param)
     return param->flags;
 }
 
+uint32_t X509_VERIFY_PARAM_get_inh_flags(const X509_VERIFY_PARAM *param)
+{
+    return param->inh_flags;
+}
+
+int X509_VERIFY_PARAM_set_inh_flags(X509_VERIFY_PARAM *param, uint32_t flags)
+{
+    param->inh_flags = flags;
+    return 1;
+}
```

Comment:
On platforms today, a long can be 32 or 64 bits depending on the compiler flags.  In order to have ABI portability across hosts with different compiler flag settings, we need to use a type that has a strictly-defined width (number of bytes).  u32int is that.  long isn't.  Since the API is correct, might as well change the field type as well, because we can, and it avoids compile issues on places where sizeof(uint32_t) != sizeof(long).  @ekasper has written about this on the team mailing list, I believe.